src/test/java/com/list/todo/init/TaskInit.java

```diff
@@ -0,0 +1,25 @@
+package com.list.todo.init;
+
+import com.list.todo.entity.Priority;
+import com.list.todo.entity.Task;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class TaskInit {
```

Comment:
You can merge TaskInit and TodoListInit classes into one with name smth like TestObjectCreationUtil